src/Features/CSharp/Portable/RemoveUnusedMembers/CSharpRemoveUnusedMembersCodeFixProvider.cs

```diff
@@ -29,5 +30,9 @@ protected override void AdjustAndAddAppropriateDeclaratorsToRemove(HashSet<Field
                 AdjustAndAddAppropriateDeclaratorsToRemove(fieldDeclarator, fieldDeclarator.Declaration.Variables, declarators);
             }
         }
+
+#if CODE_STYLE
+        protected override ISymbolDeclarationService GetSymbolDeclarationService(Document document) => CSharpSymbolDeclarationService.Instance;
+#endif
```

Comment:
really don't like this approach.  Why can't CODE_STYLE use the same discovery mechanism that roslyn uses?